File: python/week-9/exam/problem6.py
n = int(input("Enter n= "))
for i in range(1, n+1):
    for j in range(2, i+1):
        print(j, end="")
    print("1", end="")
    for k in range(i+1, n+1):
        print(k, end="")
    print()

File: python/week-9/exam/problem6.cpp
#include <stdio.h>

int main()
{
    int N, i, j;

    printf("Enter N: ");
    scanf("%d", &N);

    for(i=1; i<=N; i++)
    {
        // Print first part
        for(j=2; j<=i; j++)
        {
            printf("%d", j);
        }
        // Print second part
        printf("1");
        // Print third part
        for(j=i+1; j<= N; j++)
        {
            printf("%d", j);
        }

        printf("\n");
    }

    return 0;
}